src/activities/facebook/primary_public_location.rs

```diff
@@ -14,7 +14,7 @@ pub struct PrimaryPublicLocationObjects {
 #[allow(non_snake_case)]
 #[derive(Deserialize, Debug)]
 pub struct PrimaryPublicLocation {
-  pub primary_public_location: PrimaryPublicLocationObjects
+  pub primary_public_location_v2: PrimaryPublicLocationObjects
```

Comment:
I appreciate your work!!! `v2` is meaning a new version of the primary public location?

Reply:
Yes. If you request your data now, the format has changed and `_v2` is aappended.